src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -122,6 +122,12 @@ <h3>{% trans 'License for the content created by course participants' %}</h3>
 
     <div class="course-detail__aside">
 
+      {% if course.organization_main %}
```

Comment:
main_organization is a required field on the course so you can remove this test.